core/lib/spree/core/preference_changes_between_solidus_versions.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Spree
+  module Core
+    class PreferenceChangesBetweenSolidusVersions
```

Comment:
Could we maybe have a shorter name for this, e.g. `PreferenceDifferentiator`? Also, this could live under `Spree::Preferences` and be called `Spree::Preferences::Differentiator`. What do you think?

Reply:
Done @aldesantis!

Reply:
I think it makes sense :slightly_smiling_face: